src/functions/services/clientCredentialsService/clientCredentialsService.ts

```diff
@@ -60,6 +73,15 @@ export type IClientCredentials = {
   issuer: string;
   salt: string;
   hashed_client_secret: string;
+  redirect_uri?: string;
+};
+
+export type IRegisteredClientCredentials = {
```

Comment:
I don't understand the need to have both of these types - please may we append the redirect_uri property to the IClientCredentials type?

The redirect_uri property is mandatory in SSM, but **not** mandatory in the request body to the credential lambda

Reply:
Updated to remove this interface.